clients/client-kinesis/src/protocols/Aws_json1_1.ts

```diff
@@ -3131,6 +3137,19 @@ const deserializeAws_json1_1SubscribeToShardEventStream = (
   output: any,
   context: __SerdeContext
 ): SubscribeToShardEventStream => {
+  const targetTypes: Record<string, ParseEventStreamValidTargetTypes> = {
+    InternalFailureException: "structure",
+    KMSAccessDeniedException: "structure",
+    KMSDisabledException: "structure",
+    KMSInvalidStateException: "structure",
+    KMSNotFoundException: "structure",
+    KMSOptInRequired: "structure",
+    KMSThrottlingException: "structure",
+    ResourceInUseException: "structure",
+    ResourceNotFoundException: "structure",
+    SubscribeToShardEvent: "structure",
+  };
+  __parseEventStream(output, targetTypes);
```

Comment:
Is `parseEventStream` particular to JSON protocols? The document protocol can also be XML, calling this function will cause error then.

Reply:
it would only be included for JSON RPC, it could be called `parseEventStreamJsonRpc`